it/controllers/TimeoutsISpec.scala

```diff
@@ -0,0 +1,321 @@
+       |    },
+       |    "loginTimes": {
+       |        "currentLogin": "2021-06-07T10:52:02.594Z",
+       |        "previousLogin": null
+       |    },
+       |    "optionalCredentials": {
+       |        "providerId": "4911434741952698",
+       |        "providerType": "GovernmentGateway"
+       |    },
+       |    "authProviderId": {
+       |        "ggCredId": "xyz"
+       |    },
+       |    "externalId": "testExternalId",
+       |    "allEnrolments": [
+       |       {
+       |          "key":"HMRC-PT",
+       |          "identifiers": [
+       |             {
+       |                "key":"NINO",
+       |                "value": "$generatedNino"
+       |             }
+       |          ]
+       |       },
+       |       {
+       |          "key":"IR-SA",
+       |          "identifiers": [
+       |             {
+       |                "key":"UTR",
+       |                "value": "$generatedNino"
+       |             }
+       |          ]
+       |       }
+       |    ],
+       |    "affinityGroup": "Individual",
+       |    "credentialStrength": "strong"
+       |}
+       |""".stripMargin
+
+  private def homePageGET: Future[Result] = route(
+    app,
+    FakeRequest(GET, "/personal-account")
+      .withSession(SessionKeys.sessionId -> UUID.randomUUID().toString, SessionKeys.authToken -> "1")
+  ).get
+
+  private def getHomePageWithAllTimeouts: Future[Result] = {
+    server.stubFor(get(urlPathEqualTo(breathingSpaceUrl)).willReturn(aResponse.withFixedDelay(delayInMilliseconds)))
+    server.stubFor(get(urlEqualTo(taxComponentsUrl)).willReturn(aResponse.withFixedDelay(delayInMilliseconds)))
+    server.stubFor(get(urlPathEqualTo(taxCalcUrl)).willReturn(aResponse.withFixedDelay(delayInMilliseconds)))
+    server.stubFor(get(urlPathEqualTo(citizenDetailsUrl)).willReturn(aResponse.withFixedDelay(delayInMilliseconds)))
+    homePageGET
+  }
+
+  override def beforeEach(): Unit = {
+    Mockito.reset(mockFeatureFlagService)
+    super.beforeEach()
+    when(mockFeatureFlagService.get(ArgumentMatchers.eq(BreathingSpaceIndicatorToggle)))
+      .thenReturn(Future.successful(FeatureFlag(BreathingSpaceIndicatorToggle, isEnabled = true)))
+    when(mockFeatureFlagService.get(ArgumentMatchers.eq(TaxcalcToggle)))
+      .thenReturn(Future.successful(FeatureFlag(TaxcalcToggle, isEnabled = true)))
+    when(mockFeatureFlagService.get(ArgumentMatchers.eq(TaxComponentsToggle)))
+      .thenReturn(Future.successful(FeatureFlag(TaxComponentsToggle, isEnabled = true)))
+  }
+
+  "/personal-account" must {
+    "hide breathing space related components when breathing space connector times out" in {
+      val result            = getHomePageWithAllTimeouts
+      val content: Document = Jsoup.parse(contentAsString(result))
+      content.getElementsByClass("hmrc-caption govuk-caption-xl").get(0).text() mustBe
+        Messages("label.this.section.is") + " " + Messages("label.account_home")
```

Comment:
Could you swap out the messages calls in this file with the expected text? Makes things bit more reliable

Reply:
OK, done